pkg/printers/text.go

```diff
@@ -52,7 +53,7 @@ func (p *Text) Print(ctx context.Context, issues []result.Issue) error {
 }
 
 func (p Text) printIssue(i *result.Issue) {
-	text := p.SprintfColored(color.FgRed, "%s", i.Text)
+	text := p.SprintfColored(color.FgRed, "%s", strings.TrimSpace(i.Text)) 
```

Comment:
This change is actually because `paralleltest` returns issues with newlines at the end, causing us to not have the linter name at the end.  `paralleltest` does not have a customer issue reporter step, so I'm not sure it's wrong to make this adjustment generally.